test/e2e_node/jenkins/template.properties

```diff
@@ -1,17 +1,19 @@
 # Copy this file to your home directory and modify
-# Names of gce hosts to test against (must be resolvable) or empty (one or more of GCE_IMAGES, GCE_HOSTS is required)
+# Path to a yaml or json file describing images to run or empty
+GCE_IMAGE_CONFIG_PATH=
+# Names of gce hosts to test against (must be resolvable) or empty
 GCE_HOSTS=
-# Names of gce images to test or empty (one or more of GCE_IMAGES, GCE_HOSTS is required)
+# Comma-separated names of gce images to test or empty (one or more of GCE_IMAGE_CONFIG_PATH, GCE_IMAGES, GCE_HOSTS is required)
```

Comment:
Maybe, but right now the make workflow for local testing relies on it and there might be room for both.

We can perhaps mark it deprecated when we remove the uses of it, but otoh I'd rather set an `--images` flag than write out a yaml config for the `e2e-node-test.sh` script.

Expressive files make sense for config that's checked in, but for make options / local testing, flags make more sense.